api/controllers/calendarEntry.controller.ts

```diff
@@ -201,9 +203,22 @@ export const getCalendarEntry = async (
   _next: NextFunction,
 ) => {
   const { id } = req.params;
+  const { start } = req.query;
   try {
     const entry = await CalendarEntry.findById(id);
-    res.status(200).json(entry);
+    if (isRecurringEntry(entry)) {
+      const startDate = new Date(start as string);
+      const fiveMinBefore = fiveMinutesBefore(startDate);
+      const fiveMinAfter = fiveMinutesAfter(startDate);
+      const expandedEntry = expandRecurringEntry(
+        entry,
+        fiveMinBefore,
+        fiveMinAfter,
```

Comment:
I didn't see any context in the PR description as to why we are expanding over 10 minutes. Can you share why?

Reply:
Nit: I know there is a function called `datePlusHours`. What do you think of a function called `datePlusMinutes` to be consistent?

`datePlusMinutes(startDate, 5)`

Reply:
When expanding events, we have to give it a range. I did +/- 5 minutes arbitrarily as a small enough range to give us back a single event. It's somewhat arbitrary.

Reply:
If we were using adding/minusing minutes all over the place I think I'd be up to refactor it to `datePlusMinutes`. Since we're not I feel fine leaving the helper functions as is. With that said though if you feel strongly I'm happy to make the change. What do you prefer?

Reply:
I prefer to be consistent with how we format/modify dates. It just makes it easier to maintain. It would also be great if the way the date is modified is more or less the same across functions. So adding minutes wouldnt be so different from adding hours.

Reply:
Ok, I mostly ask because the `datePlusHours` function is on the frontend. On the backend we have other functions like `yearAfter` and `dayAfter` that are more standard about how much time they add to a date.

Reply:
Ideally, we wouldn't need a range, but from what I understand, the range is needed. Can we do +/-1 minutes instead? 10 minutes range makes it seem like we are trying to find multiple entries within that 10 minutes range, when in reality we are just trying to get one single entry.

Reply:
Sure, we can tighten the range we search for. We'll also go ahead and update the function to be `datePlusMinutes`